src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -62,10 +60,18 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
     }
 
     private HearingResponse sendCreateHearingRequest(HearingWrapper wrapper) {
-        HearingRequestPayload payload = buildHearingPayload(wrapper);
+        return hmcHearingApi.createHearingRequest(
+            idamService.getIdamTokens().getIdamOauth2Token(),
+            idamService.getIdamTokens().getServiceAuthorization(),
+            buildHearingPayload(wrapper));
+    }
 
-        return hmcHearingApi.createHearingRequest(idamService.getIdamTokens().getIdamOauth2Token(),
-            idamService.getIdamTokens().getServiceAuthorization(), payload);
+    private void sendUpdateHearingRequest(HearingWrapper wrapper) {
+        hmcHearingApi.updateHearingRequest(
+            idamService.getIdamTokens().getIdamOauth2Token(),
+            idamService.getIdamTokens().getServiceAuthorization(),
+            wrapper.getOriginalCaseData().getSchedulingAndListingFields().getActiveHearingId().toString(),
```

Comment:
Can you put some validation in here?
Also, can you use getUpdatedCaseData instead?

Reply:
Perhaps, put a method in Mapping?